### PR TITLE
Put the configuration item description into the configuration file and fix the bug where the heartbeat still reports plugin information after the plugin is uninstalled.

### PR DESCRIPTION
**Pull request by luanwenfei-venus**
【Fix issue】#1323
[Modification content] 1. Add configuration description to the configuration file 2. Fix the bug where the heartbeat still reports plug-in information after the plug-in is uninstalled
[Use case description] Not involved
[Self-test situation] 1. The static check has been cleared. 2. The self-verification has passed. After the plug-in is uninstalled, it will be cleared in the Sermant Backend plug-in display area.
[Scope of Impact] 1. The configuration description has been deleted from the official document. The configuration file shall be subject to the configuration in the project file.

### CODE REVIEW

File: sermant-agentcore/sermant-agentcore-config/config/bootstrap.properties [hunk 1]
```diff
@@ -1,4 +1,8 @@
+# 指定当前Sermant制品的制品名，用于在多Sermant场景中标识不同的Sermant制品，基于artifact隔离不同Sermant的配置及服务治理能力
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-config/config/bootstrap.properties:1]: 制品的制品名？这是业界的常用词吗

File: sermant-agentcore/sermant-agentcore-config/config/plugins.yaml [hunk 2]
```diff
@@ -11,12 +16,15 @@ plugins:
...
   - service-visibility
   - tag-transmission
-# agentmain启动时生效，配置支持动态安装插件，active类型插件将会主动启用, passive类型插件需通过指令或接口调用启用，允许卸载
+# dynamicPlugins用于配置支持动态安装的插件, agentmain启动时生效, 允许卸载:
+#  1. active类型插件将会主动启用
+#  2. passive类型插件需通过指令或接口调用启用
 dynamicPlugins:
   active:
-#    - active-plugin
+  #    - active-plugin
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-config/config/plugins.yaml]: 和“#    - passive-plugin”对齐

File: sermant-agentcore/sermant-agentcore-config/config/config.properties [hunk 3]
```diff
@@ -1,61 +1,96 @@
...
 dynamic.config.connectRetryTimes=5
+# 指定在启动Sermant时连接配置中心的超时时间（ms）
 dynamic.config.connectTimeout=1000
+# 指定是否开启配置中心授权, 开启后需验证用户名密码
+dynamic.config.enableAuth=false
+# 指定连接动态配置中心时的用户名
 dynamic.config.userName=
+# 指定连接动态配置中心时的密码
 dynamic.config.password=
+# 指定为密码进行加解密的密钥
```
Comment (TangLeDaily) [inline sermant-agentcore/sermant-agentcore-config/config/config.properties]: 这个秘钥对用户名和密码都进行加密和解密

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/common/PluginSchemaValidator.java [hunk 4]
```diff
@@ -53,21 +53,31 @@ public static Map<String, String> getPluginVersionMap() {
...
+
     /**
      * 检查名称和版本
      *
-     * @param name    插件名称
+     * @param name 插件名称
      * @param jarFile 插件包
      * @return 为真时经过名称和版本校验，为插件包或插件服务包，为假时表示第三方jar包
      * @throws IOException 获取manifest文件异常
+     * @throws SchemaException 传入插件名和从资源文件中检索到的不一致
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/common/PluginSchemaValidator.java:80]: 我看就抛出了 IOexception啊

File: sermant-agentcore/sermant-agentcore-config/config/config.properties [hunk 5]
```diff
@@ -1,61 +1,96 @@
...
+#=============================元数据配置================================#
+# 指定应用名称, 用于服务注册等服务治理场景
 service.meta.application=default
+# 指定服务版本, 用于服务注册、标签路由等服务治理场景
 service.meta.version=1.0.0
+# 指定服务命名空间, 用于服务注册等服务治理场景
 service.meta.project=default
+# 指定服务所在环境, 用于服务注册等服务治理场景
 service.meta.environment=
+# 指定服务所在az, 用于服务注册、标签路由等服务治理场景
```
Comment (TangLeDaily) [inline sermant-agentcore/sermant-agentcore-config/config/config.properties]: az还是不要写简称，与文档对应

File: sermant-agentcore/sermant-agentcore-config/config/plugins.yaml [hunk 6]
```diff
@@ -1,4 +1,9 @@
-# premain启动时生效，配置支持静态安装的插件，不允许卸载
+# 插件的加载顺序如下：
+#   1. 首先按照`plugins`中配置的插件顺序来加载默认插件。
+#   2. 然后再按照`profile`配置的场景顺序来加载场景插件列表, 各场景的插件加载顺序也和配置文件中的顺序一致。
+#   3. 如果`profiles`中的配置的插件已经在之前加载过, 则不再重复加载。
+#   4. 不同插件的相同拦截点的字节码增强的生效顺序与插件加载顺序是一致的。
+# plugins用于配置静态插件, premain启动时生效, 配置支持静态安装的插件, 不允许卸载
```
Comment (TangLeDaily) [inline sermant-agentcore/sermant-agentcore-config/config/plugins.yaml:7]: 建议加上注释premain启动时，只有plugins生效，dynamicPlugins不生效。反之也是

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/common/PluginSchemaValidator.java (hunk 4 above)
Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/common/PluginSchemaValidator.java:80]: throw new SchemaException(SchemaException.UNEXPECTED_NAME, nameAttr.toString(), name);

Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/common/PluginSchemaValidator.java:80]: throw new SchemaException(SchemaException.UNEXPECTED_NAME, nameAttr.toString(), name);

File: sermant-agentcore/sermant-agentcore-config/config/bootstrap.properties (hunk 1 above)
Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-config/config/bootstrap.properties:1]: 采用mavne依赖中的artifactId的称呼

File: sermant-agentcore/sermant-agentcore-config/config/plugins.yaml (hunk 2 above)
Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-config/config/plugins.yaml]: 已修改

File: sermant-agentcore/sermant-agentcore-config/config/config.properties (hunk 3 above)
Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-config/config/config.properties]: 已修改

File: sermant-agentcore/sermant-agentcore-config/config/config.properties (hunk 5 above)
Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-config/config/config.properties]: 已修改

File: sermant-agentcore/sermant-agentcore-config/config/plugins.yaml (hunk 6 above)
Comment (luanwenfei-venus) [reply sermant-agentcore/sermant-agentcore-config/config/plugins.yaml:7]: 已修改

**Review by daizhenyu — APPROVED** [review]

**Review by TangLeDaily — APPROVED** [review]

**Pull request merged**